internal/distro/fedora30/distro.go

```diff
@@ -129,7 +135,8 @@ func New(confPaths []string) *Fedora30 {
 		},
 		KernelOptions: "ro biosdevname=0 net.ifnames=0",
 		Bootable:      false,
-		Assembler:     func(uefi bool) *pipeline.Assembler { return r.rawFSAssembler("filesystem.img") },
+		DefaultSize:   3221225472,
```

Comment:
`3 * GigaByte` ?

Reply:
Don't know how to comment on the commit message, so leaving this here: please flesh it out to make it clear what you are adding (allowing size to be passed in) and changing (per output-type default sizes (as opposed to?), for when no size is passed in).

Reply:
> `3 * GigaByte` ?

This seems to be a go-ism. I've seen this done for time and kind of liked the idea: https://gobyexample.com/timeouts